sdk/src/genesis_block.rs

```diff
@@ -147,6 +147,10 @@ impl Builder {
         self.genesis_block.inflation = inflation;
         self
     }
+    pub fn rent_calculator(mut self, rent_calculator: RentCalculator) -> Self {
```

Comment:
I'd prefer to see the rent sysvar moved from a member variable into the accounts vector...